Foundation/NSHTTPCookie.swift

```diff
@@ -277,21 +294,27 @@ public class NSHTTPCookie : NSObject {
         Version 1 maps to RFC2965 cookies. There may be future versions.
         @result the version of the receiver.
     */
-    public var version: Int { NSUnimplemented() }
+    public var version: Int {
+        return _properties[NSHTTPCookieVersion]
+    }
     
     /*!
         @method name
         @abstract Returns the name of the receiver.
         @result the name of the receiver.
     */
-    public var name: String { NSUnimplemented() }
+    public var name: String {
+        return _properties[NSHTTPCookieName]
```

Comment:
Actually, I think what we want to do here is keep the _properties dictionary holding a Swift.String object instead of NSString.